test/mri/tests/ruby/test_fiber.rb

```diff
@@ -79,12 +79,12 @@ def test_error
       f.resume
       f.resume
     }
-    assert_raise(RuntimeError){
-      Fiber.new{
-        @c = callcc{|c| @c = c}
-      }.resume
-      @c.call # cross fiber callcc
-    }
+    # assert_raise(RuntimeError){
```

Comment:
Can this be handled a different way? I'm not wild about commenting out code in an upstream test.

Reply:
I'm afraid not, and that's a main problem with MRI tests.
We can exclude the method but then we would ignore the other 7 useful assertions.

We could try passing that test I guess.

Reply:
We once had some minimal support for continuations in Java (b60b386522bb5f5d81bd695b411d1d8cd56a0bdd) but I'm not willing to restore that much logic.
Maybe we can have a minimal pure-Ruby implementation though.

Reply:
406c65a62d7b50c932f28ac3e89bd911ce499601 adds this minimal support, what do you think?
It doesn't seem harmful to define those and just forbid the part we cannot easily implement, that is the jump back in time by `call`-ing the continuation.